content/en/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -42,17 +42,17 @@ kubelet [flags]
     <tr>
       <td></td><td style="line-height: 130%; word-wrap: break-word;">If true, adds the file directory to the header</td>
     </tr>
-    
```

Comment:
@furukawa3 , this file is auto-generated. I would not bother to remove the white space in this file.

Reply:
I see.  So, is it better to omit  under *en/docs/reference/*  files ??